cmake/CMakeLists.txt

```diff
@@ -1524,6 +1527,15 @@ if (onnxruntime_BUILD_CSHARP)
   include(onnxruntime_csharp.cmake)
 endif()
 
+if (onnxruntime_BUILD_WEBASSEMBLY)
+  if (CMAKE_BUILD_TYPE STREQUAL "Debug" OR onnxruntime_ENABLE_WEBASSEMBLY_TEST)
```

Comment:
should we keep consistency for Debug and Release on whether enabling exception catching?

Reply:
as default behavior of emscripten is disabling exception, I think it's useful to provide more debugging information from debug build.

Reply:
I suggest to reuse build flag '--disable_exceptions' and at the same time keep the consistency of Release/RelWithDebInfo/Debug.

Reply:
Do you suggest that WebAssembly builds with enabling exceptions by default?